PSReadLine.build.ps1

```diff
@@ -138,10 +94,27 @@ Synopsis: Run the unit tests
 #>
 task RunTests BuildMainModule, BuildXUnitTests, { Start-TestRun -Configuration $Configuration -Framework $Framework }
 
+<#
+Synopsis: Check if the help content is in sync.
+#>
+task CheckHelpContent {
+    if ($CheckHelpContent) {
+        # This step loads the dll that was just built, so only do that in another process
+        # so the file isn't locked in any way for the rest of the build.
+        $psExePath = Get-PSExePath
+        & $psExePath -NoProfile -NonInteractive -File $PSScriptRoot/tools/CheckHelp.ps1 $Configuration
```

Comment:
The CheckHelp script is not checking that the key bindings match between the code and docs.